README.md

```diff
@@ -21,3 +21,14 @@ A Repository to collect and showcases developers' favorite resources to learn a
     - What didn't you understand?
     
   ---
+
+
+### [Eric Davidson](https://github.com/erdavids)
+
+> Eric is software engineer that is passionate about creating mobile applications, generative art, and tutorials that benefits others.
+
+#### Git
+1. [Learning Git Branching](https://learngitbranching.js.org)
+    - This is a fantastic way to learn about Git branching because of the physical interaction. The site lets you visualize exactly how your commands are affecting the project.
+    - The interactivity of the project is what helped it work for me. It is a lot easier to understand something when you're able to actually use it and see the effects.
+    - For complete beginners it might be difficult to translate the lessons from this site directly to a real project, especially one where you are collaborating with other people.
```

Comment:
Hey, In that case, can you add something which you would recommend for complete beginners